docs/README.md

```diff
@@ -441,6 +441,90 @@ You can expand any of the messages that matches the filter to see the full stack
+![image](./images/user-guide/37-services-overview.png)  
+
+#### Services SLOs  
+
+The *predefined SLOs* are also deployed as part of [Terraform code](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring/04_slos.tf) and currently are for the mentioned custom services, the Istio service and Rating service.  
+
+**Custom services SLOs**
+``` 
+Custom service availability SLO: 90% of all non-4XX requests within the past 30 day windowed period return with 200 OK status 
+```
+
+``` 
+Custom service Latency SLO:  90% of requests that return 200 OK responses return in under 500 ms 
+```  
+To view the exiting SLOs, in the Services Overview screen choose the desired service.
+
+*For example for checkoutservice*:
+
+![image](./images/user-guide/47-choose-checkout-custom-service.png)
+
+![image](./images/user-guide/36-checkoutservice-overview.png)
+
+**Additional predefined SLOs:**
+
+```
+Istio service availability SLO: 90% of all non-4XX requests within the past 30 day windowed period return with 200 OK status
+```
+```
+Istio service latency SLO: 99% of requests that return 200 OK responses return in under 500 ms
+```
+```
+Rating service availability SLO: 99% of all non-4XX requests within the past 30 day windowed period  return with 200 OK status
+```
+```
+Rating service latency SLO: 99% of requests that return 200 OK responses return in under 175 ms
+```
+```
+Rating service's data freshness SLO: during a day 99.9% of minutes have at least 1 successful recollect API call
+```
+
+#### Configure your own SLIs and SLOs
+
+> **Remember**  Our scope to examine and measure our users' happiness is User journey, so in order to create the SLO you need to identify the most important ones to the business. Then we want to *identify the metrics* that are closest to the customer experience and ingest that data.  
+
+You can [configure your own SLIs and SLOs](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/ui/create-slo) for an existing service or for your own custom service.  
+
+#### Let's demonstrate that using the checkout auto-defined Istio service:
+1. In the service screen we will choose Create SLO:
+![image](./images/user-guide/39-checkout-service.png)
+2.Then we will set our SLI, we need to choose SLI type and the method(request vs window based):
+![image](./images/user-guide/42-checkoutservice-sli.png)
+3. Then we wil define our metric and we can also preview it's performance based on historical data:
+![image](./images/user-guide/41-checkoutservice-define-sli.png)
+4. Then we will configure our SLO, our target in a specific time window. We can also choose between [rolling window or a calendar window](https://sre.google/workbook/implementing-slos/):
+![image](./images/user-guide/43-set-slo.png)
+
+#### Configure Burn Rate Alerts
+
+After you created the SLO, you can create[Burn Rate Alerts](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/alerting-on-budget-burn-rate)for those.
```

Comment:
Nit: change to `After you create the SLO, create [Burn Rate Alerts]... for them`